lightbulb/decorators.py

```diff
@@ -98,13 +98,17 @@ def command(
         inherit_checks (:obj:`bool`): Whether or not the command should inherit checks from the parent group. Only
             affects subcommands. Defaults to ``False``.
         pass_options (:obj:`bool`): Whether the command will have its options passed as keyword arguments when invoked.
+        name_localizations (Mapping[Union[hikari.locales.Locale, :obj:`str`], :obj:`str`]): A set of name localizations for this command. Defaults to an empty dictionary.
```

Comment:
Incorrect type in docstring. Should be:
`Mapping[Union[:obj:`hikari.locales.Locale`, :obj:`str`], :obj:`str`]
Change to the above in all places, and change the argument description to something more appropriate as mentioned above.

Reply:
Fixed in 05b489b88d4a2d8dc10a91c1a74c531fe867aacc.